test/globals.d.ts

```diff
@@ -0,0 +1,2 @@
+declare module 'relay-local-schema';
```

Comment:
lol i should publish types for this maybe

Reply:
or just use ts everywhere :P